js/stdlib/ts/ALGO.ts

```diff
@@ -1978,9 +1997,19 @@ export const connectAccount = async (networkAccount: NetworkAccount): Promise<Ac
           return await viewMapRef(mapi, a);
         },
       };
+
```

Comment:
You can actually define the cache right here and it will be specialized for THIS call to `contract`. Scope, man. It's what's for dinner.

Reply:
Yes, I would have done that, but from what I could tell, the places I needed to invalidate from were outside of that scope.  If that's not the case, I'll move it back in.